viewer.js

```diff
@@ -3,7 +3,7 @@ var Renderer = require('./renderer.js');
 
 var viewerHtml = require('./viewer.html');
 
-var Viewer = Component('kokorigami-viewer', HTMLElement, {
+var Viewer = Component('kokorigami-viewer', {
   _: {
```

Comment:
The data should probably be an object since we want to parse it.

The _ attribute doesn't work for this case since it's not meant to be writable.